charts/incubator/alfresco-dbp/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: alfresco-dbp
-version: 0.8.4
+version: 0.8.4-andreif
```

Comment:
The version should be `1.5` as per this [comment](https://issues.alfresco.com/jira/browse/AUTH-345?focusedCommentId=632370&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-632370).